src/export/fribbels.rs

```diff
@@ -96,13 +95,48 @@ impl OptimizerExporter {
     }
 
     pub fn add_characters(&mut self, characters: GetAvatarDataScRsp) {
-        let mut characters: Vec<Character> = characters.avatar_list.iter()
-            .filter(|a| a.base_avatar_id < 8000) // skip trailblazer, handled in `write_hero`
+        let (characters, multipath_characters) = characters.avatar_list.iter()
+            .partition::<Vec<_>, _>(|a| MultiPathAvatarType::from_i32(a.base_avatar_id as i32).is_none() );
+
+        let mut characters: Vec<Character> = characters.iter()
             .filter_map(|char| export_proto_character(&self.database, char))
             .collect();
 
         info!(num=characters.len(), "found characters");
         self.characters.append(&mut characters);
+
+        info!(num=multipath_characters.len(), "found multipath base avatars");
+        self.multipath_base_avatars.extend(multipath_characters.into_iter().map(|c| (c.base_avatar_id, c.clone())));
+    }
+
+    pub fn add_multipath_characters(&mut self, characters: GetMultiPathAvatarInfoScRsp) {
+        let mut characters: Vec<Character> = characters.multi_path_avatar_info_list.iter()
+            .filter_map(|char| export_proto_multipath_character(&self.database, char))
+            .collect();
+
+        // Try to find a trailblazer to determine the gender
+        if let Some(trailblazer) = characters.iter().find(|c| c.name == "Trailblazer") {
```

Comment:
this will conflict with https://github.com/IceDynamix/reliquary-archiver/issues/43, maybe `c.id >= 8000` is better?